assets/differentialabundance_report.Rmd

```diff
@@ -765,28 +765,39 @@ for (i in 1:nrow(contrasts)){
     cat("\n##### ", pvt, " p values\n")
     pval_column <- p_value_types[[pvt]]
   
-    full_de$differential_status <- FALSE
-    full_de$differential_status[abs(full_de[[params$differential_fc_column]]) > log2(params$differential_min_fold_change) & full_de[[pval_column]] < p_value_thresholds[[pvt]]] <- TRUE
-
+    
+    de_fc <- abs(full_de[[params$differential_fc_column]]) >= log2(params$differential_min_fold_change)
+    de_fc_label <- paste("abs(logFC) >=", params$differential_min_fold_change)
+    
+    de_pval <- full_de[[pval_column]] <= p_value_thresholds[[pvt]]
+    de_pval_label <- paste(pvt, "<=", p_value_thresholds[[pvt]])
+    
+    full_de$differential_status <- "Not significant"
+    full_de$differential_status[de_fc] <- de_fc_label
+    full_de$differential_status[de_pval] <- de_pval_label
+    full_de$differential_status[de_fc & de_pval] <- paste(de_fc_label, '&', de_pval_label)
+    full_de$differential_status <- factor(full_de$differential_status, levels = c("Not significant", de_fc_label, de_pval_label, paste(de_fc_label, '&', de_pval_label)), ordered = TRUE) # Factorize status so that non-significant is always first
```

Comment:
```suggestion
    de_pval <- full_de[[pval_column]] <= p_value_thresholds[[pvt]]
    de_pval_label <- paste(pvt, "<=", p_value_thresholds[[pvt]])
    
    de_pval_fc_label <- paste(de_fc_label, '&', de_pval_label)
    
    full_de$differential_status <- "Not significant"
    full_de$differential_status[de_fc] <- de_fc_label
    full_de$differential_status[de_pval] <- de_pval_label
    full_de$differential_status[de_fc & de_pval] <- de_pval_fc_label
    full_de$differential_status <- factor(full_de$differential_status, levels = c("Not significant", de_fc_label, de_pval_label, de_pval_fc_label), ordered = TRUE) # Factorize status so that non-significant is always first
```